pages/dkp/konvoy/2.2/dkp-upgrade/upgrade-konvoy/essential/index.md

```diff
@@ -58,13 +58,28 @@ dkp upgrade capi-components
 
 The command should output something similar to the following:
 
-```text
+```sh
 ✓ Upgrading CAPI components
 ✓ Waiting for CAPI components to be upgraded
 ✓ Initializing new CAPI components
 ✓ Deleting Outdated Global ClusterResourceSets
 ```
 
+<p class="message--warning"><strong>WARNING:</strong> If you upgraded 1.8 to 2.1 before upgrading to 2.2, run the following upgrade commands instead.</p>
```

Comment:
Instead of what?

Reply:
Oh, so, this is if a user: 
upgrades from 1.8 -> 2.1, and then you need to upgrade from 2.1 -> 2.2
   This issue will happen where you need this fix

If you upgrade with a fresh build of 2.1 (never upgraded) -> 2.2 
    You won't encounter this issue.

Reply:
We need to rework this a bit.   This new sequence is an alternate sequence for the procedure described on line 53-67.

I suggest we change lines 53-67 into a sequence of steps, something like this:

1.   (If you cluster was upgraded to 2.1 from 1.8)  Prepare the old cert-manager installation for removal:

 helm -n cert-manager get manifest cert-manager-kubeaddons | kubectl label -f - clusterctl.cluster.x-k8s.io/core=cert-manager
kubectl delete validatingwebhookconfigurations/cert-manager-kubeaddons-webhook mutatingwebhookconfigurations/cert-manager-kubeaddons-webhook

2.  (for all clusters) Upgrade capi-components:

dkp upgrade capi-components

...output sequence

3. (If your cluster was upgraded to 2.1 from 1.8) Remove the old cert-manager you marked in step 1:

helm -n cert-manager delete cert-manager-kubeaddons

Reply:
Moved this suggestion to Release Notes

Reply:
Took Craig suggestions to RN as requested.  Removed from Upgrade Docs